airbyte-api/src/main/openapi/config.yaml

```diff
@@ -895,21 +967,15 @@ components:
           items:
             $ref: "#/components/schemas/SourceRead"
     # SOURCE SPECIFICATION
-    SourceSpecificationId:
-      type: string
-      format: uuid
     SourceSpecification:
       description: The specification for what values are required to configure the source.
       example: { user: { type: string } }
     SourceSpecificationRead:
```

Comment:
will this now read from the spec method on each docker image?

Reply:
That is the idea, but it doesn't have to happen in strict lockstep with this change.